doc/release/1.13.0-notes.rst

```diff
@@ -189,6 +189,16 @@ if all fields that are non-masked in both are equal, thus making the behaviour
 identical to what one gets by comparing an unstructured masked array and then
 doing ``.all()`` over some axis.
 
+np.matrix with booleans elements can now be created using the string syntax
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+np.matrix failed whenever one attempts to use it with booleans, e.g.,
+``np.matrix('True')``. This failed because the previous algorithm looked in
+the alphabet ``0123456789.-+jeEL`` to decode each string passed to
```

Comment:
`np.matrix` probably wants backticks too, while you're fixing that (doesn't matter too much though, because the release notes get a final formatting pass anyway)